next/components/sections/ContactsSection.tsx

```diff
@@ -18,20 +19,40 @@ const ContactsSection = ({
 
   return (
     <Section {...rest}>
-      <div
-        className={cx('flex flex-col gap-4 bg-white p-6', {
-          'border border-border': border,
-        })}
-      >
-        {filteredContacts.map((contact, index) => (
-          <Contact
-            // eslint-disable-next-line react/no-array-index-key
-            key={index}
-            contact={contact}
-            className="border-b border-border pb-4 last:border-0 last:pb-0"
-          />
-        ))}
-      </div>
+      {layout === 'condensed' && (
```

Comment:
Maybe use Strapi generated enum?

Reply:
I'm not sure about advantages, suggestions in IDE works also for string and I find them more readable. But we can use enums if it's better :)
Not changing in this PR.